apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -71,28 +72,47 @@ const configuredUiItems = new Map<string, UiItem>([
             {
               id: ModelsTreeComponent.id,
               getLabel: ModelsTreeComponent.getLabel,
-              render: () => (
-                <ModelsTreeComponent
+              render: () => {
```

Comment:
Does changing arrow function to something like `render: function ModelsTreeComponentRenderer() {` resolves eslint error?

Reply:
Yes, using your suggestion did resolve the eslint error, however after refactoring it was no longer necessary.